packages/mds-db/geographies.ts

```diff
@@ -128,13 +145,16 @@ export async function editGeography(geography: Geography) {
     }
   })
   const sql = `UPDATE ${schema.TABLE.geographies} SET ${conditions} WHERE geography_id='${geography.geography_id}' AND publish_date IS NULL`
+
   await client.query(sql, vals.values())
-  return geography
+  const res = await readSingleGeography(geography.geography_id)
```

Comment:
Nit: why not
```
return await readSingleGeography(geography.geography_id)
```

Reply:
`return await` is redundant and will probably be flagged by the linter. You can just `return readSingleGeography(...)` though if you prefer.